juju/docs
# Remove media/ directory

Link to assets.ubuntu.com for all images, and remove the `media/` folder.

Update instructions in `README.md` and `srv/en/contributing.md`.

Now, to add images they should be uploaded at https://manager.assets.ubuntu.com, or to any other image host like https://imgur.com/, and then linked to directly from the Markdown documents.

Fixes https://github.com/ubuntudesign/base-squad/issues/306

QA
--

`./run`, and go to http://localhost:8205, check the site and the images look fine.
